editor/components/inserter/inline-menu.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { pick } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { getInlineBlocks } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import InserterGroup from './group';
+
+export default class InserterInlineMenu extends Component {
+	render() {
+		const inlineBlocks = getInlineBlocks();
+		const items = inlineBlocks.map( ( inlineBlock ) => (
+			pick( inlineBlock, [ 'id', 'title', 'icon' ] )
```

Comment:
I don't understand why we need to pick properties here. 🤔 Seems like a waste of energy?

Reply:
You're right, that's unnecessary. Removed in https://github.com/WordPress/gutenberg/pull/5794/commits/f84cb62b0b604eae12a1a6a9aa8623de9e65764a.